appveyor.yml

```diff
@@ -148,8 +148,8 @@ test_script:
   - sh src/ci/run.sh
 
 on_failure:
-  - cat %CD%\sccache.log
-  - cat C:\Users\appveyor\AppData\Local\Temp\1\build-cache-logs\*.log
+  - cat %CD%\sccache.log || exit 0
+  - cat C:\Users\appveyor\AppData\Local\Temp\1\build-cache-logs\*.log || exit 0
```

Comment:
To be honest I'm relying on the ar.exe errors to let me read this log (if it rarely executes because the ar.exe errors vanish, I wouldn't see that as the end of the world!). I'll probably remove it again once it's all fixed.